### PR TITLE
Remove GetByNamespace since it's redundant.

### PR DESCRIPTION
**Pull request by miaoyq**

### CODE REVIEW

**Comment by mfojtik** [comment]
nice catch!

[merge]

File: pkg/client/imagestreams.go [hunk 1]
```diff
@@ -71,7 +71,7 @@ func (c *imageStreams) Get(name string) (result *imageapi.ImageStream, err error
 // GetByNamespace returns information about a particular image stream in a particular namespace and error if one occurs.
 func (c *imageStreams) GetByNamespace(namespace, name string) (result *imageapi.ImageStream, err error) {
 	result = &imageapi.ImageStream{}
-	c.r.Get().Namespace(namespace).Resource("imageStreams").Name(name).Do().Into(result)
+	err = c.r.Get().Namespace(namespace).Resource("imageStreams").Name(name).Do().Into(result)
```
Comment (mfojtik) [inline pkg/client/imagestreams.go]: while you are here, can you please refactor this to not use the named result variables?

Comment (kargakis) [inline pkg/client/imagestreams.go]: GetByNamespace doesn't seem to be used anywhere so I would suggest removing it altogether.

Comment (miaoyq) [inline pkg/client/imagestreams.go]: Thanks! @mfojtik @kargakis 
Other methods in the module also use the named result variables. Does they need to be refactored? @mfojtik

**Comment by miaoyq** [comment]
@mfojtik @kargakis  /cc   Thanks a lot!

**Comment by kargakis** [comment]
@miaoyq just remove GetByNamespace since it's redundant.

**Comment by miaoyq** [comment]
I have removed `GetByNamespace`  @kargakis

**Comment by kargakis** [comment]
LGTM [merge]

**Pull request merged**